src/constants.ts

```diff
@@ -68,3 +68,6 @@ export enum ScmType {
 export const publishTaskId: string = 'publish';
 export const installExtensionsId: string = 'installExtensions';
 export const funcPackId: string = 'funcPack';
+
+export const WEBSITE_CONTENTAZUREFILECONNECTIONSTRING: string = 'WEBSITE_CONTENTAZUREFILECONNECTIONSTRING';
```

Comment:
Why are these in the constants file? I've mentioned this before - but I don't like throwing every constant in here unless it's actually used by multiple files or if it's a notable/important constant.